Makefile

```diff
@@ -23,6 +23,11 @@ build-in-docker:
 		$(NAME)-build make -e deps build
 	docker rmi $(NAME)-build || true
 
+clean:
+	rm -rf build/*
+	@$(QUIET) docker rm $(shell docker ps -aq) || true
+	@$(QUIET) docker rmi herokuish:dev || true
+
```

Comment:
Copy pasta from another project. It tells make to not print out the command. It can be removed.